### DISCUSSION TITLE
Exporting Orange nomograms

### DISCUSSION THREAD

**Discussion by csurytd** [biolab/orange3#6375] [General]
Hi,

I was wondering if it's possible to export an Orange nomogram in an interactive way, i.e not as an image. Any ideas?

**Comment by janezd** [comment]
You can run the widget without Orange. If you just run ownomogram.py (from a suitable environment) you'll see it shows a "preview" nomogram. If you can provide it with proper data, it will work. It would require just a few lines of code. This code here https://github.com/biolab/orange3/blob/master/Orange/widgets/visualize/ownomogram.py#L1382 loads some data, creates naive Bayesian classifier and shows it in nomogram.

Doing this however requires having (most of) Orange installed.

**Reply by csurytd** [reply]
Great, thanks! I will try it soon.
